presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -34,6 +34,7 @@
 import io.prestosql.plugin.hive.metastore.PartitionFilter;
 import io.prestosql.plugin.hive.metastore.PartitionWithStatistics;
 import io.prestosql.plugin.hive.metastore.PrincipalPrivileges;
+import io.prestosql.plugin.hive.metastore.RecordingHiveMetastore.TablesWithParameterCacheKey;
```

Comment:
Please move `TablesWithParameterCacheKey` to top level class, just as we do for `PartitionFilter`, `HiveTableName`, etc

Reply:
Thanks, fixed